core/src/main/java/com/segment/analytics/kotlin/core/platform/plugins/StartupQueue.kt

```diff
@@ -40,7 +41,7 @@ class StartupQueue : Plugin, Subscriber {
 
     override fun execute(event: BaseEvent): BaseEvent? {
         if (!started.get()) {
-            analytics.log("SegmentStartupQueue queueing event", event = event)
+            analytics.log("SegmentStartupQueue queueing event")
```

Comment:
why not log the event?

Reply:
analytics.log is all about a simple log message. Logging the event would use the history option. Event is a ton of metadata when we're just trying to capture certain elements they are free to pass in.